src/app.js

```diff
@@ -2,0 +1,51 @@
+'use strict';
+
+/* eslint-disable no-console */
+
+const fs = require('node:fs');
+const path = require('node:path');
+
+const moveFile = (source, destination) => {
+  if (!fs.existsSync(source)) {
+    console.error(`Error: Source file does not exist: ${source}`);
+    return;
+  }
+
+  if (source === destination) {
+    return;
+  }
+
+  let finalDestination = destination;
+
+  if (
+    (fs.existsSync(destination) && fs.lstatSync(destination).isDirectory()) ||
+    destination.endsWith('/')
+  ) {
+    finalDestination = path.join(destination, path.basename(source));
+  }
+
+  try {
+    const fileData = fs.readFileSync(source, 'utf8');
+
+    try {
+      fs.writeFileSync(finalDestination, fileData, 'utf8');
+      fs.rmSync(source, { force: true });
```

Comment:
Using `fs.rmSync` with `{ force: true }` is potentially dangerous as it will remove the file without any checks. Consider adding a confirmation step or ensuring that the file is indeed the one intended for deletion.